fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -523,13 +524,15 @@ protected void updateOauthSchema() {
                 buildOAuthSchemaModel(pdm);
                 updateSchema(pdm);
 
-                // Apply privileges if asked
-                if (grantTo != null) {
-                    grantPrivilegesForOAuth();
+                if (this.exitStatus == EXIT_OK) {
+                    // Apply privileges if asked
+                    if (grantTo != null) {
+                        grantPrivilegesForOAuth();
+                    }
+    
+                    // Mark the schema as up-to-date
+                    svm.updateSchemaVersion();
```

Comment:
I'm sorry I didn't see this before.  

What happens when someone runs an older version than the current one, would this ever end up with an out of sync version? It seems to insert without checking the version number.